src/content/docs/en/reference/api-reference.mdx

```diff
@@ -2302,6 +2302,39 @@ The `code` property accepts human-readable versions of all HTTP status codes. Th
 
 The `message` property accepts a string. (e.g. "User must be logged in.")
 
+## Environment variables (`astro:env`)
+
+<p>
+<Since v="5.0.0" />
+</p>
+
+The `astro:env` API lets you configure a type-safe schema for environment variables you have set. This allows you to indicate whether they should be available on the server or the client, and define their data type and additional properties. For examples and usage instructions, [see the `astro:env` guide](/en/guides/environment-variables/#type-safe-environment-variables).
+
+### `getSecret()`
+
+<p>
+<Since v="5.0.0" />
+</p>
+
+The `getSecret()` helper function can be used for several things:
+
+1. **Get the raw value of a secret defined in your schema**: if you define a variable `FOO` of type boolean, using `getSecret()` will allow you to get the value as string:
+
+   ```js
+   import {
+      FOO, // boolean
+      getSecret
+   } from 'astro:env/server'
+
+   getSecret('FOO') // string | undefined
+   ```
+
+2. **Retrieve secrets not specified in your schema**: that can be useful if you need to get a secret depending on dynamic data, eg. a database or API.
+
+   :::note
```

Comment:
I think the word "use" is doing a lot of heavy lifting here. At first, I read it as "if you have environment variables at all" which is why I had trouble making sense of it. I think now you mean, "if you have to build logic around your environment variables" or "use your environment variables programmatically?"

Can you be more explicit about "use" (because *having* them in a project is also "using" them, and my first read was, "If I'm supposed to use getSecret if I have environment variables, what was all the other stuff about?)

Also, is it possible that this really doesn't require a note, but just a line somewhere in the description/explanation as an "advantage of `astro:env`? :smile: 

e.g.

> The `getSecret()` helper function can fetch the value of your environment variables even if you switch adapters, and therefore is more flexible than using `process.env`